.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,23 @@
+name: Update Breaking Change Status
+on:
+  pull_request:
+    types:
+      - labeled
+jobs:
+  add-comment:
+    if: github.event.label.id == '4598495472'
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      statuses: write
+    steps:
+    - uses: actions/checkout@v2
+    - name: Allow breaking change to merged
+      uses: Sibz/github-status-action@v1
```

Comment:
Are there alternatives to taking a dependency on an arbitrary community plugin here?

Reply:
I'll investigate!

Reply:
go/github-actions and go/github-docs/security-recommendations are relevant here

Reply:
looks like this was changed